### DISCUSSION TITLE
How can I allign and height adjust images with qmd?

### DISCUSSION THREAD

**Discussion by kurianbenoy** [quarto-dev/quarto-cli#4389] [Q&A] [answered]
In quarto markdown, there is option to :

adjust width
![Elephant](elephant.png){width=80%}

adjust figure alignment

![Elephant](elephant.png){fig-align="left"}

How can I adjust both width and do figure alignment. I couldn't find anything in docs[1]

[1] https://quarto.org/docs/authoring/figures.html

**Comment by cscheid** [comment, marked as answer]
Just combine the two options

```
![a sized right-aligned kitten](kitten.jpg){width="80%" fig-align="right"}
```

**Reply by cscheid** [reply]
This is now documented in [markdown basics](https://github.com/quarto-dev/quarto-web/blob/main/docs/authoring/markdown-basics.qmd), (but wasn't until now! Thanks for the feedback.)

**Reply by kurianbenoy** [reply]
Thank you for clarifying.

This is the doc URL for [Markdown basics](https://quarto.org/docs/authoring/markdown-basics.html)
